test/apps/iodemo/ucx_wrapper.cc

```diff
@@ -674,7 +639,18 @@ void UcxContext::destroy_worker()
 
     ucp_worker_destroy(_worker);
     if (_use_epoll) {
-        _epoll.del(_worker_fd);
+        memset(&ev, 0, sizeof(ev));
+        ev.events  = EPOLLIN;
+        ev.data.fd = _worker_fd;
+
+        if (epoll_ctl(_epoll_fd, EPOLL_CTL_DEL, _worker_fd, &ev) == -1) {
+            UCX_LOG << "epoll del worker fd: " << _worker_fd << " failed.";
+        }
```

Comment:
do we really need this? can just destroy the epoll fd

Reply:
@yosefe 
The epoll manual page says:
```When no longer required, the file descriptor returned by epoll_create() should be closed by using close(2).``` **When all file descriptors referring to an epoll instance  have  been  closed,  the kernel destroys the instance and releases the associated resources for reuse.**
Should we also delete the monitored fd from the epoll instance?